pallets/inflation-manager/src/lib.rs

```diff
@@ -35,7 +35,7 @@ use peaq_primitives_xcm::Balance;
 use sp_runtime::{traits::BlockNumberProvider, Perbill};
 use sp_std::cmp::Ordering;
 
-pub const BLOCKS_PER_YEAR: peaq_primitives_xcm::BlockNumber = 365 * 24 * 60 * 60 / 12_u32;
```

Comment:
This reminds me - This works for peaq, but for Krest, we'll have to set the BlockRewards and DoRecalculationAt storage values to appropriate values also with a storage migration to the increased number of blocks per year to keep tokenomics consistent over the years.

Reply:
Ohhh, that's important. When we apply the delayed TGE improvement on the Krest network, we'll add the set delay TGE extrinsic. Therefore, after introducing the async backing there, we can use that extrinsic to set the delay TGE time. Let me add it into the checklist